src/test/java/org/opentripplanner/gtfs/mapping/StopTimeMapperTest.java

```diff
@@ -132,7 +167,9 @@ public void testMap() {
 
   @Test
   public void testMapWithNulls() {
-    org.opentripplanner.model.StopTime result = subject.map(new StopTime());
+    var st = new StopTime();
+    st.setStop(buildStop());
+    org.opentripplanner.model.StopTime result = subject.map(st);
```

Comment:
This still needs to be updated

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/5564/commits/40fcc74beaec4eb36c6cfe01ffd64fe7fdd13f24